cpp/src/Ice/TraceUtil.cpp

```diff
@@ -26,11 +26,7 @@ using namespace IceInternal;
 static void
 printIdentityFacetOperation(ostream& s, InputStream& stream)
 {
-    ToStringMode toStringMode = ToStringMode::Unicode;
-    if (stream.instance())
-    {
-        toStringMode = stream.instance()->toStringMode();
-    }
+    ToStringMode toStringMode = stream.instance()->toStringMode();
```

Comment:
This null check is no longer required with the latest InputStream updates and instance is always provided.